### ISSUE TITLE
changelog?

### ISSUE THREAD

**Issue by sckott** [randy3k/rtichoke#51]
would it be possible to include even a minimal changelog of changes for each tagged release?

**Comment by randy3k** [comment]
Certainly. I am working on a major refactoring. Will certain be more transparent after v0.2

**Comment by sckott** [comment]
cool, thanks
